.buildkite/pipeline.yml

```diff
@@ -0,0 +1,61 @@
+steps:
+  - label: "build-gnu-x86"
+    commands:
+     - cargo build --release
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+    plugins:
+      - docker#v3.0.1:
+          image: "fandree/rust-vmm-dev"
+          always-pull: true
+          
+  - label: "style"
+    command: cargo fmt --all -- --check
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+    plugins:
+      - docker#v3.0.1:
+          image: "fandree/rust-vmm-dev"
+          always-pull: true
+  
+  - label: "unittests-gnu-x86"
+    commands:
+     - cargo test
```

Comment:
In particular this is where the bzImage could be prepared.

Reply:
@rbradford @andreeaflorescu Just want to confirm, to fix this issue, bzImage should be added to the server running CI test for rust-vmm crate, and specified under .buildkite/pipeline.yml as Rob mentioned. if my understanding is correct, will Andrea help put it or should I do?

Reply:
@bjzhjing I cannot help you cause you misspelled my name :cry:

Reply:
I was thinking about asking around if someone knows a better way to use a bzImage binary in tests. It sounds absurd to be so complicated and I am sure that other people had this problem as well.

In any case, if there is no other way but having it build as a pre-step on the buildkite agent, I can work with you to make it happened. I think though that whoever writes the code also owns the testing, so I can provide info and access to the machine that is running the CI and you can do your own thing there. Would that be ok?

Reply:
I don't think the binary should be manually place on the build machine as then it makes it hard to reproduce. A simple script that either downloads a prebuilt kernel rpm/deb and extracts the kernel should be added to the repository and run as a step prior to the unit test running itself. This also means all CI configuration is stored in revision control.

In the meantime I propose that we merge a version without bzimage support and spin the commit adding that into it's own PR.

Reply:
@andreeaflorescu Hi Andreea, sorry for the typo of your name 

> @bjzhjing I cannot help you cause you misspelled my name 😢

Hi Andreea, I'm really sorry for making the typo, please forgive me! :-P

Reply:
@rbradford @andreeaflorescu I'm considering to add feature for Elf and bzImage to separate the test by define cargo test --features in the yaml file. If you both agree with that, I will make the change. 

And then, I think I need to get an account of buildkite, I'd like to see how to define an upload action for a pre-compiled bzImage in the yaml, and download it to linux-loader crate for test running. How do you think?

Reply:
@bjzhjing I was trolling you with the name, sorry! Sounds good to me to have it as a separate feature. For the buildkite account, please ping on email/slack and I'll give you access to the rust-vmm one.